src/XliffTasks/Model/XlfDocument.cs

```diff
@@ -155,6 +155,26 @@ public bool Update(TranslatableDocument sourceDocument, string sourceDocumentId)
                     changed = true;
                 }
 
+                if (sourceDocument is ResxDocument)
```

Comment:
This class shouldn't know what a resx is. If we want to limit it to only some formats, then we could have a virtual propery on TranslatableDocument. 

But I wonder if we can't just do this for all formats since:

1. Most documents are resx in practice, so I don't think perf helps.

2. I don't think we'd ever have a string in any translatable document with "{N}" where we expect translators to translate to anything other than "{N}".

Reply:
Fixed.